wger/nutrition/models/meal.py

```diff
@@ -50,6 +50,13 @@ class Meta:
         editable=False,
         on_delete=models.CASCADE,
     )
+    log_item = models.ForeignKey(
+        LogItem,
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="meals"
+    )
```

Comment:
The foreign key should be in the logs, sorry if it wasn't very clear. Then in  _logMealPlan you can simply set the current meal when creating the LogItem instance

Reply:
ah okay okay. sorry my bad i'll get it sorted